enterprise_access/apps/content_assignments/tests/factories.py

```diff
@@ -47,4 +47,4 @@ class Meta:
     lms_user_id = factory.LazyAttribute(lambda _: FAKER.pyint())
     content_key = factory.LazyAttribute(lambda _: random_content_key())
     content_title = factory.LazyAttribute(lambda _: f'{FAKER.word()}: a master class')
-    content_quantity = factory.LazyAttribute(lambda _: FAKER.pyint())
+    content_quantity = -1
```

Comment:
nit: I wonder if its worth keeping the random int, but change it up to generate negatives?